src/scripts/StoreTalismanApi.js

```diff
@@ -72,4 +83,111 @@ export default class StoreTalismanApi extends StoreRestApi {
 		delete this.axiosInstance.defaults.headers.Authorization;
 		await this.axiosInstance.get('/oauth/logout');
 	}
+
+	async getAllProjects() {
+		this.setSitemapPath();
+		const queryData = {
+			query: GET_ALL_PROJECTS_QUERY,
+			variables: {
+				limit: 100,
+				sortField: 'title',
+				sortDirection: 'ascending',
+			},
+		};
+		return this.axiosInstance.post('/graphql', queryData);
+	}
+
+	async getAllSitemaps(projectId) {
+		if (!projectId) {
+			projectId = this.sitemapsPath.split('/')[2];
+		}
+		this.setSitemapPath(projectId);
+		const queryData = {
+			query: GET_ALL_PROJECT_SITEMAPS_QUERY,
+			variables: {
+				filterSettings: {
+					projects: [projectId],
+					crawlersTypes: ['SitemapCrawlers'],
+				},
+				sortField: 'title',
+				sortDirection: 'ascending',
+			},
+		};
+		const sitemaps_obj = await this.axiosInstance.post('/graphql', queryData);
+		return sitemaps_obj.data.data.paginationCrawler.listCrawler.map(sitemap =>
+			JSON.parse(sitemap.sitemap)
+		);
+	}
```

Comment:
- why gql?
- where is limit
- restore sitemap migrations

Reply:
fix